delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/CompositeViews.scala

```diff
@@ -219,9 +219,28 @@ final class CompositeViews private (
       id: IdSegment,
       project: ProjectRef,
       rev: Long
+  )(implicit subject: Subject): IO[CompositeViewRejection, ViewResource] =
+    deprecate(id, project, notDeprecatedOrDeletedWithEventQuotas, rev)
+
+  /**
+    * Deprecate a view without any extra checks on the projects API.
+    * @see [[deprecate(id, project, rev)]]
+    */
+  private[compositeviews] def deprecateWithoutProjectChecks(
```

Comment:
The name is quite long, maybe something like `forceDeprecate` would be shorter?

Reply:
Yes It would be shorter but less explicit. It won't forceDeprecate since it can still return a rejection for other reasons (related to the state machine)